style.css

```diff
@@ -603,15 +551,18 @@ button:disabled {
 .my-stack {
   display: flex;
   flex-direction: column;
-  margin-top: 150px;
-  margin-bottom: 120px;
+  justify-content: center;
   align-items: center;
   color: #fff;
+  padding-top: 120px;
+  padding-bottom: 10%;
+  padding-left: 0;
   gap: 24px;
 }
 
```

Comment:
- [ ] Good job for adding the technologies section as [suggested](https://github.com/allenkays/Portfolio-setup-and-mobile-first/pull/4#discussion_r1264574899) 🥇🥇 However, it is not looking close to the Figma template. Kindly make sure that after adding the section, it looks close to the Figma template as much as possible  ⛏🤓

![image](https://github.com/allenkays/Portfolio-setup-and-mobile-first/assets/106119105/c97bd671-e496-4438-8bc2-a2e1a08c8897)